develop/plone/searching_and_indexing/indexing.rst

```diff
@@ -281,64 +281,53 @@ The difference with metadata is that
 
 * You store always a value copy as is
 
-To create metadata colums in your ``catalog.xml`` add::
+To create metadata colums in your ``catalog.xml`` add:
 
-	<?xml version="1.0"?>
-	<object name="portal_catalog" meta_type="Plone Catalog Tool">
-
-		<!-- Add a new metadata column which will read from context.getSignificant() function -->
-		<column value="getSignificant"/>
+.. code-block:: xml
 
-	</object>
+    <?xml version="1.0"?>
+    <object name="portal_catalog" meta_type="Plone Catalog Tool">
+      <!-- Add a new metadata column which will read from context.getSignificant() function -->
+      <column value="getSignificant"/>
+    </object>
 
 
 When indexing happens and how to reindex manually
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Content item reindexing is run when
-
-Plone calls reindexObject() if
+Content indexing happens automatically if:
 
 * The object is modified by the user using the standard edit forms
 
 * portal_catalog rebuild is run (from *Advanced* tab)
 
-* If you add a new
-  index you need to run :doc:`Rebuild catalog </develop/plone/searching_and_indexing/catalog>`
-  to get the existing values from content objects to new index.
-
-* You might also want to call :doc:`reindexObject()
```

Comment:
no, that paragraph is completely useless; before saying you're _uncomfortable deleting material_ you have to review if the documentation in question is useful; what's the use case of that? none, zero, nada...